doc/ap_verify/index.rst

```diff
@@ -14,7 +14,7 @@ Repository
    https://github.com/lsst-dm/ap_verify
 
 Jira component
-   `ap_verify <https://jira.lsstcorp.org/issues/?jql=project %3D DM AND component %3D ap_verify>`_
+   `ap_verify <https://jira.lsstcorp.org/issues/?jql=project %3D DM %20AND%20 component %3D ap_verify>`_
```

Comment:
@mrawls it turns out I wasn't quite right in recommending spaces in URLs -- browsers can handle spaces, but Sphinx deletes them when writing the HTML.

Reply:
Well there's a fun mess. There may be a broken link lurking in ap_pipe too, then, but I'm not going to lose sleep over it.